### PR TITLE
KM review (part 2) -- Editorial pass on Section 12

### PR DESCRIPTION
**Pull request by thomas-fossati**
https://mailarchive.ietf.org/arch/msg/rats/7-UJAqHPMLhrGCk9tyh3JqGu8rw/

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -990,13 +990,21 @@ an Attestation Result for any other purpose.
...
+
+* end-to-end encryption,
+* end-to-end integrity protection,
+* replay attack prevention, 
+* denial of service protection,
+* authentication, 
+* authorization,
+* fine grained access controls, and
+* logging in line with current threat models and zero trust architectures.
+
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1007]:
```suggestion
Any solution that conveys information used for security purposes, whether
such information is in the form of Evidence, Attestation Results,
Endorsements, or Appraisal Policy must support
the security properties of confidentiality, integrity, and availability.  
A conveyance protocol includes the typical transport security considerations:

* end-to-end encryption,
* end-to-end integrity protection,
* replay attack prevention, 
* denial of service protection,
* authentication, 
* authorization,
* fine grained access controls, and
* logging in line with current threat models and zero trust architectures.

```

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -992,11 +992,19 @@ an Attestation Result for any other purpose.
 
 Any solution that conveys information used for security purposes, whether
 such information is in the form of Evidence, Attestation Results,
-Endorsements, or Appraisal Policy, needs to support end-to-end integrity protection
-and replay attack prevention, and often also needs to support additional
-security protections.  For example, additional means of authentication,
-confidentiality, integrity, replay, denial of service and privacy
-protection are needed in many use cases.
+Endorsements, or Appraisal Policy must support
+the security properties of confidentiality, integrity, and availability.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:996]:
```suggestion
confidentiality, integrity, and availability.  
```

**Comment by mcr** [comment]
Try to revise this entirely, as it seems to change the set of requirements on conveyance.
There are definitely problems in the old text.

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -992,11 +992,19 @@ an Attestation Result for any other purpose.
...
+A conveyance protocol includes the typical transport security considerations:
+
+* end-to-end encryption,
+* end-to-end integrity protection,
+* replay attack prevention, 
+* denial of service protection,
+* authentication, 
+* authorization,
+* fine grained access controls, and
+* logging in line with current threat models and zero trust architectures.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1006]:
```suggestion
Endorsements, or Appraisal Policy must support end-to-end integrity protection
and replay attack prevention, and often also needs to support additional
security properties, including:

* end-to-end encryption,
* denial of service protection,
* authentication,
* fine grained access controls, and
* logging in line with current threat models and zero trust architectures.
```

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -992,11 +992,16 @@ an Attestation Result for any other purpose.
...
-security protections.  For example, additional means of authentication,
-confidentiality, integrity, replay, denial of service and privacy
-protection are needed in many use cases.
+security properties, including:
+
+* end-to-end encryption,
+* denial of service protection,
+* authentication,
+* fine grained access controls, and
+* logging in line with current threat models and zero trust architectures.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1003]:
```suggestion
* logging.
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -992,11 +992,16 @@ an Attestation Result for any other purpose.
...
+Endorsements, or Appraisal Policy must support end-to-end integrity protection
 and replay attack prevention, and often also needs to support additional
-security protections.  For example, additional means of authentication,
-confidentiality, integrity, replay, denial of service and privacy
-protection are needed in many use cases.
+security properties, including:
+
+* end-to-end encryption,
+* denial of service protection,
+* authentication,
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1001]:
```suggestion
* authentication,
* auditing,
```

File: draft-ietf-rats-architecture.md [hunk 6]
```diff
@@ -998,11 +998,17 @@ an Attestation Result for any other purpose.
 
 Any solution that conveys information used for security purposes, whether
 such information is in the form of Evidence, Attestation Results,
-Endorsements, or Appraisal Policy, needs to support end-to-end integrity protection
+Endorsements, or Appraisal Policy must support end-to-end integrity protection
 and replay attack prevention, and often also needs to support additional
-security protections.  For example, additional means of authentication,
-confidentiality, integrity, replay, denial of service and privacy
-protection are needed in many use cases.
```
Comment (nedmsmith) [inline draft-ietf-rats-architecture.md:1005]: The "example" restates the list of 'must' be supported security properties. This seems redundant and/or isn't a real example. Might be better to say "must support end-to-end integrity and replay attack prevention, and often needs to support authentication, confidentiality, denial-of-service and privacy protection."

**Pull request merged**